12/mysql-tomcat/xwiki/docker-entrypoint.sh

```diff
@@ -112,8 +112,8 @@ function configure() {
   file_env 'DB_PASSWORD' 'xwiki'
   file_env 'DB_HOST' 'db'
   file_env 'DB_DATABASE' 'xwiki'
-  file_env 'INDEX_HOST' 'localhost'
-  file_env 'INDEX_PORT' '8983'
+  file_env 'INDEX_BASEURL' 'http://localhost:8983/solr/xwiki'
+  file_env 'INDEX_TYPE' 'embedded'
```

Comment:
I don't really understand why we need "INDEX_TYPE". Could INDEX_BASEURL be an empty string by default and use that as criteria for example ?

Reply:
Actually forget that I'm fine with that since it's simply about following the standard configuration and someone might introduce a new type at some point.